bill/payment.go

```diff
@@ -9,8 +9,8 @@ import (
 
 // Payment contains details as to how the invoice should be paid.
 type Payment struct {
-	// The party responsible for paying for the invoice, if not the customer.
-	Payer *org.Party `json:"payer,omitempty" jsonschema:"title=Payer"`
+	// The party responsible for receiving payment of the invoice, if not the supplier.
+	Payee *org.Party `json:"payee,omitempty" jsonschema:"title=Payer"`
```

Comment:
Turned out that the "Payee" is the most relevant detail here as opposed to who the payer should be.